### PR TITLE
Fir error - unbound variable

### PR DESCRIPTION
**Pull request by pmb59**
The new version of Snakemake running in SLURM complains about

`generic_routines.sh: line 189: fixesFile: unbound variable`

this PR should solve it.

### CODE REVIEW

File: generic_routines.sh [hunk 1]
```diff
@@ -186,7 +186,7 @@ applyFixes() {
     fixesFilePath=$2
     fileTypeToBeFixed=$3
 
-    echo "NOTE: Fix will not be applied in lines of $fixesFile that miss a tab character"
+    echo "NOTE: Fix will not be applied in lines of fixesFile that miss a tab character"
```
Comment (anilthanki) [inline generic_routines.sh]: Won't this just print fixesFile instead of value of `$fixesFile`?

Comment (pmb59) [reply generic_routines.sh]: The var `$fixesFile` is never defined nor used. Probably we carried it over from previous PRs, without noticing so far as it was not producing error

**Review by anilthanki — APPROVED** [review]
LGTM

**Comment by pmb59** [comment]
@anilthanki Thanks for reviewing

**Pull request merged**